addons/mail/views/discuss_channel_views.xml

```diff
@@ -52,6 +52,9 @@
                             <field name="active" invisible="1"/>
                             <field name="channel_type" groups="base.group_no_one"/>
                             <field name="description" placeholder="Topics discussed in this group..." readonly="not is_editable"/>
+                            <field name="parent_channel_id" readonly="True" groups="base.group_no_one"/>
+                            <field name="sub_channel_ids" readonly="True" groups="base.group_no_one"/>
+                            <field name="from_message_id" readonly="True" groups="base.group_no_one"/>
```

Comment:
Should have write override to enforce them read-only + add tests to cover it.

And maybe have them readonly on the field definition directly too?

Reply:
Write+test+readonly on field done